cli/features/tutorial.feature

```diff
@@ -28,7 +28,7 @@ Feature: Following the tutorial
       | Name of the application to create | todo-app           |
       | Description                       | A todo application |
       | Initial version                   |                    |
-      | ExoCom version                    | 0.16.0             |
+      | ExoCom version                    | latest             |
```

Comment:
Nevermind, `latest` could mean the user pulled the `latest` from 3 weeks ago. I'll change this back to a hard version number